flight/targets/revolution/fw/pios_board.c

```diff
@@ -515,14 +534,54 @@ void PIOS_Board_Init(void) {
 			hwRevoMini.MaxChannel, hwRevoMini.CoordID, 1);
 #endif /* PIOS_INCLUDE_RFM22B */
 
+	PIOS_SENSORS_Init();
 
-	if (PIOS_I2C_Init(&pios_i2c_mag_pressure_adapter_id, &pios_i2c_mag_pressure_adapter_cfg)) {
-		PIOS_DEBUG_Assert(0);
+#if defined(PIOS_INCLUDE_HMC5883)
+	PIOS_WDG_Clear();
+
+	uint8_t Magnetometer;
```

Comment:
Our variable naming standard is lowercase with underscores separating words. I know some old stuff doesn't follow this but we should be following it for all new code, and updating old stuff when we touch it.